.github/workflows/check-nix-format.yml

```
@@ -83,7 +83,7 @@ jobs:
 
           if (( "${#unformattedFiles[@]}" > 0 )); then
             echo "Some new/changed Nix files are not properly formatted"
-            echo "Please go to the Nixpkgs root directory, run \`nix-shell\`, then:"
+            echo "Please format them using the Nixpkgs-specific \`nixfmt\` by going to the Nixpkgs root directory, running \`nix-shell\`, then:"
```

If this were single-quoted, I think the backticks wouldn't need escaping anymore.
```suggestion
            echo 'Please format them using the Nixpkgs-specific `nixfmt` by going to the Nixpkgs root directory, running `nix-shell`, then:'
```

Not important though, if you'd rather consistently use double-quotes.